app/presenters/account_show_presenter.rb

```diff
@@ -34,10 +36,36 @@ def show_gpo_partial?
     user.gpo_verification_pending_profile?
   end
 
+  def show_ipp_partial?
+    user.pending_in_person_enrollment.present?
+  end
+
   def showing_any_partials?
+    show_password_reset_partial? ||
+      show_gpo_partial? || show_ipp_partial? || show_pii_partial?
+  end
+
+  def showing_alerts?
     show_service_provider_continue_partial? ||
-      show_password_reset_partial? ||
-      show_gpo_partial?
+      show_password_reset_partial?
+  end
+
+  def show_unverified?
+    show_password_reset_partial? &&
+      (!show_ipp_partial? || !show_gpo_partial?)
+  end
+
+  def service_provider_or_app_name
+    if user.identities.count == 0
+      APP_NAME
+    else
+      user.identities.last.friendly_name
```

Comment:
Do we have any guarantees that this is the name of the service provider that the user was trying to access for identity verification, as opposed to another partner application that the user is authenticated with (at IAL1) ?